internal/am/job_tracker.go

```diff
@@ -115,9 +117,36 @@ func filterSavedJobs(jobs []amclient.Job, saved map[string]struct{}) []amclient.
 // ConvertJobToPreservationTask converts an amclient.Job to a
 // package_.PreservationTask.
 func ConvertJobToPreservationTask(job amclient.Job) package_.PreservationTask {
+	st, co := getJobTimestamp(job)
 	return package_.PreservationTask{
-		TaskID: job.ID,
-		Name:   job.Name,
-		Status: jobStatusToPreservationTaskStatus[job.Status],
+		TaskID:      job.ID,
+		Name:        job.Name,
+		Status:      jobStatusToPreservationTaskStatus[job.Status],
+		StartedAt:   st,
+		CompletedAt: co,
 	}
 }
+
+func getJobTimestamp(job amclient.Job) (sql.NullTime, sql.NullTime) {
```

Comment:
`getJobTimestamp` can be misleading because it implies that the function returns a single timestamp. A better name could be `jobTimeRange`. I've omitted the "get" prefix because of this recommendation: https://go.dev/doc/effective_go#Getters.